api/views.py

```diff
@@ -337,9 +336,16 @@ def create(self, request, **kwargs):
                         },
                         status=400,
                     )
-                # the post m2m hook will add tag
-                puzzle.metas.add(meta)
+                else:
```

Comment:
This else is not necessary, no?

Reply:
No, but I think an explicit else statement is easier to read.

Reply:
Fair enough.